cmd/maya-apiserver/cstor-operator/spc/controller.go

```diff
@@ -167,6 +171,10 @@ func (c *Controller) updateSpc(oldSpc, newSpc interface{}) {
 		runtime.HandleError(fmt.Errorf("Couldn't get spc object %#v", newSpc))
 		return
 	}
+	if spc.Labels[string(apis.OpenEBSUpgradeKey)] == "true" {
```

Comment:
logic should be handled at informer code

Reply:
Same as above comment